cmd/backend-api/main.go

```
@@ -172,7 +208,39 @@ func main() {
+		} else {
+			health["queue_status"] = "disabled"
+		}
+		
+		// Set appropriate HTTP status
+		statusCode := http.StatusOK
+		if health["status"] == "degraded" {
+			statusCode = http.StatusServiceUnavailable
+		}
+		
+		w.WriteHeader(statusCode)
+		json.NewEncoder(w).Encode(health)
```

golangci-lint (1.64.8)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)